pkgs/servers/knxd/default.nix

```diff
@@ -35,8 +36,12 @@ stdenv.mkDerivation rec {
     ++ lib.optional withUsb libusb1
     ++ lib.optional stdenv.isDarwin argp-standalone;
 
-  configureFlags = lib.optional (!withSystemd) "--disable-systemd"
-    ++ lib.optional (!withUsb) "--disable-usb";
+  configureFlags = [
+    (lib.enableFeature withSystemd "systemd")
+    (lib.enableFeature withUsb "usb")
+  ];
+
+  CC_FOR_BUILD="${buildPackages.stdenv.cc}/bin/cc";
```

Comment:
Is it possible to use `makeFlags = [ "CC=cc" ];`? Maybe it's not needed to specify a path for this, as C compiler is already included in stdenv.

Reply:
But `${buildPackages.stdenv.cc}/bin/cc` is a host compiler (what is needed here), while `cc` is a target one.